app/editor/src/features/content/list-view/components/tool-bar/filter/InputOption.tsx

```diff
@@ -40,11 +40,9 @@ export const InputOption: React.FC<any> = ({
     style,
   };
 
-  /** ensures that the element is unchecked in the menu when the item is removed from the selection */
+  /** ensures that the element is checked/unchecked in the menu when the item is added/removed from the selection */
   React.useEffect(() => {
-    if (!isSelected) {
-      (document.getElementById(children) as HTMLInputElement).checked = false;
-    }
+    (document.getElementById(children) as HTMLInputElement).checked = isSelected;
```

Comment:
using the DOM isn't ideal.  I wonder why this was implemented this way originally?

Reply:
@Fosol good call, I didn't really question it. I just pushed some changes that is an alternative, more minimalist way. I tested as best I could and don't see any weird side effects. Let me know if you want me to revert it back to my initial commit.